build.json

```diff
@@ -216,6 +216,7 @@
       ],
       "src": [
         "src/core/census/grpc_context.c",
+        "src/core/channel/census_filter.c",
```

Comment:
ctiller@, what is the preferred location for filters like this - in the channel directory (current) or in package specific directories (census in this case). I think I'd prefer it in census, but not strongly.